app/presenters/return-requirements/check/returns-requirements.presenter.js

```diff
@@ -39,6 +47,32 @@ function _abstractionPeriod (abstractionPeriod) {
   return `From ${startDate} to ${endDate}`
 }
 
+function _agreementsExceptions (agreementsExceptions) {
+  if (agreementsExceptions[0] === agreementsExceptionsText.none) {
+    return 'None'
+  }
+
+  const formattedExceptions = agreementsExceptions.map((exception) => {
+    return agreementsExceptionsText[exception]
+  })
+
+  if (formattedExceptions.length === 1) {
+    return formattedExceptions[0]
+  } else if (formattedExceptions.length === 2) {
+    return formattedExceptions.join(' and ')
+  } else {
+    return _formatExceptionsToSerialCommaList(formattedExceptions)
+  }
+}
+
+function _formatExceptionsToSerialCommaList (exceptions) {
+  return exceptions
+    .slice(0, exceptions.length - 1)
+    .join(', ') +
+    (exceptions.length > 2 ? ',' : '') +
+    ' and ' + exceptions[exceptions.length - 1]
+}
+
```

Comment:
We're only calling this function if there are more than 2 `agreementExceptions`. Which means I think we can ditch the `exceptions.length > 2` check.

That makes things simple enough I'd have no reservations about moving this

```javascript
  return formattedExceptions
    .slice(0, formattedExceptions.length - 1)
    .join(', ') +
    ' and ' + formattedExceptions[formattedExceptions.length - 1]
```

into `_agreementsExceptions()` and dropping the function altogether.

Reply:
`(exceptions.length > 2 ? ',' : '')` this adds the serial comma at the end of the join so there is a comma before the and.

Reply:
i can remove condition. and just have a + ','

Reply:
I'll just add the comma in the and condition. No idea why i didn't do that in the first placee

```
exceptions.slice(0, exceptions.length - 1).join(', ') + ', and ' + exceptions[exceptions.length - 1]
```